From: David Rabel <david.rabel@noresoft.com>
Date: Wed, 8 Feb 2017 22:52:01 +0100
Subject: Delete extra COPYING file in data/

---
 data/Makefile.am | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/data/Makefile.am b/data/Makefile.am
index 4876934..ceae183 100644
--- a/data/Makefile.am
+++ b/data/Makefile.am
@@ -5,8 +5,7 @@
 # Install from data/ folder
 thisdatadir = $(pkgdatadir)
 thisdata_DATA = *.html \
-                *.png \
-                COPYING
+                *.png
 
 # Install audio-recorder.desktop file
 desktopdir = $(datadir)/applications
