From: David Rabel <david.rabel@noresoft.com>
Date: Sat, 18 Feb 2017 12:50:39 +0100
Subject: Missing copyright notice added

---
 src/gst-recorder.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/gst-recorder.c b/src/gst-recorder.c
index 01bd957..21ff8d0 100644
--- a/src/gst-recorder.c
+++ b/src/gst-recorder.c
@@ -1,4 +1,5 @@
 /*
+ * Copyright (c) 2002 Iain Holmes.
  * Copyright (c) Team audio-recorder.
  *
  * This library is free software; you can redistribute it and/or
